From ebb654b6ff1bc0c7ccab5e212a42c1ed41286d71 Mon Sep 17 00:00:00 2001
From: Harald Sitter <sitter@kde.org>
Date: Wed, 11 May 2016 12:34:36 +0200
Subject: [PATCH] don't list mimetype field it produces false positive on
 linting

(mimetype field would require Exec= to have placeholder codes)
---
 desktop/org.kde.spectacle.desktop | 1 -
 1 file changed, 1 deletion(-)

diff --git a/desktop/org.kde.spectacle.desktop b/desktop/org.kde.spectacle.desktop
index 83df40b..6b2a43d 100644
--- a/desktop/org.kde.spectacle.desktop
+++ b/desktop/org.kde.spectacle.desktop
@@ -57,7 +57,6 @@ Name[x-test]=xxSpectaclexx
 Name[zh_CN]=Spectacle
 Name[zh_TW]=Spectacle
 Categories=Qt;KDE;Utility;
-MimeType=
 Exec=spectacle
 Icon=spectacle
 Type=Application
-- 
2.7.4

